app/modules/sightings/models.py

```diff
@@ -1608,42 +1608,42 @@ def _ensure_annot_data_in_response(self, annot, response):
         individual = Individual.query.get(individual_guid) if individual_guid else None
 
         if annot.guid not in response['annotation_data'].keys():
-            encounter_location = (
-                annot.encounter.get_location_id() if annot.encounter else None
-            )
             # add annot data
             image_url = url_for(
                 'api.sightings_sighting_id_result_annotation_src_asset',
                 sighting_guid=self.guid,
                 annotation_guid=annot.guid,
                 _external=True,
             )
+            encounter = annot.encounter
+            sighting = annot.get_sighting()
             response['annotation_data'][str(annot.guid)] = {
                 'viewpoint': annot.viewpoint,
-                'encounter_location': encounter_location,
-                'individual_guid': str(individual_guid),
+                'encounter_location': encounter.get_location_id() if encounter else None,
```

Comment:
technically we dont need to verify `if encounter` here as it bails at the very top of this `if not annot.encounter....`

Reply:
Fixed